test/integration/instance.test.js

```diff
@@ -685,5 +685,104 @@ describe(Support.getTestDialectTeaser('Instance'), () => {
         expect(user.username).to.equal('Peter');
       });
     });
+
+    it('supports custom deletedAt field', function() {
```

Comment:
`restore` tests are still in here. Happy to move them to their own file if you want.